debian/control

```diff
@@ -252,7 +252,13 @@ Package: eos-tech-support
 Architecture: all
 Depends: ${misc:Depends}, ${eos:Depends}
 Description: EndlessOS Technical Support Tools
- This is a collection technical support tools that are installed by default
+ This is a collection of technical support tools that are installed by default
  with the EndlessOS operating system.  They are not required as part of the
  normal user experience but are relatively small and helpful for advanced
  troubleshooting.
+
+Package: eos-tech-support-iso
```

Comment:
That's OK, I agree. It's not ideal (if we're shipping tools, we should actually provide the things they need to run), but in the short term this makes sense. I just wanted to make sure I understood the purpose of the separate metapackage.